go.work.sum

```diff
@@ -1,3 +1,5 @@
+bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
I think it is a bit weird that this has grown this much from just a deps version bump. Should we look into this?